suites/apis/rasCronjobValidate.js

```diff
@@ -0,0 +1,24 @@
+Feature('RAS cronjob validation');
+
+const { expect } = require('chai');
+const { interactive, ifInteractive } = require('../../utils/interactive.js');
+
+const TARGET_ENVIRONMENT = process.env.GEN3_COMMONS_HOSTNAME;
+
+Scenario('Validate fence-visa-update job @manual', ifInteractive(
+  async () => {
+    const result = await interactive(`
+            1. Get a new passport from RAS /userinfo endpoint
+            2. Log into ${TARGET_ENVIRONMENT} with RAS Login (RAS user and password)
+            3. Check ga4gh_visa_v1 table by command 'gen3 psql fence'
+                - number of visas for the user
+                - check the JTI and expiration on the visa
```

Comment:
Maybe we should specify that the JTI/expiration check should be on the latest visa. Also, I don't think it would hurt to document a SQL query for this step (e.g. `SELECT ga4gh_visa, expires FROM ga4gh_visa_v1 ORDER BY expires DESC LIMIT 1;`)

Reply:
i have added the SQL queries on the both step 3 and step 6 so that it helps the user